lib/editor/util/__tests__/__snapshots__/map.js.snap

```diff
@@ -834,96 +665,40 @@ Object {
     ],
     Array [
       Array [
-        -77.92183700000001,
-        42.086513000000004,
-      ],
-      Array [
-        -77.92188300000001,
-        42.086956,
-      ],
-      Array [
-        -77.921929,
-        42.087368,
-      ],
-      Array [
-        -77.922043,
-        42.088386,
-      ],
-      Array [
-        -77.922112,
-        42.088844,
+        -77.92278528213501,
+        42.09424190611456,
       ],
       Array [
-        -77.922135,
-        42.089099,
+        -77.92253851890564,
+        42.0923709824232,
       ],
     ],
     Array [
       Array [
-        -77.922135,
-        42.089099,
+        -77.92253851890564,
+        42.0923709824232,
       ],
       Array [
-        -77.92231799999999,
-        42.090522,
-      ],
-      Array [
-        -77.92235600000001,
-        42.090847000000004,
-      ],
-      Array [
-        -77.922432,
-        42.091491000000005,
-      ],
-      Array [
-        -77.922509,
-        42.092201,
-      ],
-      Array [
-        -77.922654,
-        42.093502,
-      ],
-      Array [
-        -77.922684,
-        42.093745999999996,
+        -77.9222059249878,
+        42.08925796155427,
       ],
     ],
     Array [
       Array [
-        -77.922684,
-        42.093745999999996,
-      ],
-      Array [
-        -77.922707,
-        42.09391,
+        -77.9222059249878,
+        42.08925796155427,
       ],
       Array [
-        -77.92274499999999,
-        42.094158,
-      ],
-      Array [
-        -77.922799,
-        42.094528000000004,
-      ],
-      Array [
-        -77.922974,
-        42.095657,
-      ],
-      Array [
-        -77.923104,
-        42.096481,
+        -77.92187333106995,
+        42.08646327816015,
```

Comment:
This coordinate in the snapshot for `should update the first stop` seems out of place. This indicates that the shape goes backwards.

Here are the pattern coordinates and stops before the test:

<img width="348" alt="Screen Shot 2019-04-03 at 2 53 02 PM" src="https://user-images.githubusercontent.com/3112493/55516311-b6113000-5621-11e9-8646-32548a38b371.png">
 
And after recalcuate shape is ran:

<img width="356" alt="Screen Shot 2019-04-03 at 2 53 13 PM" src="https://user-images.githubusercontent.com/3112493/55516320-bd383e00-5621-11e9-8da6-0c17de066e3e.png">

The segment between the first and 2nd stop looks fine, but the shape doesn't continue to the last stop. Instead it goes backwards. You can't see if from that last picture, so here is a more zoomed in look:

<img width="653" alt="Screen Shot 2019-04-03 at 3 04 43 PM" src="https://user-images.githubusercontent.com/3112493/55516382-e1941a80-5621-11e9-9c69-1f9095c34152.png">